src/model/remoteConfig.ts

```diff
@@ -62,6 +66,7 @@ interface RainbowConfig extends Record<string, string | boolean | number> {
   op_chains_tx_enabled: boolean;
   mainnet_tx_enabled: boolean;
   goerli_tx_enabled: boolean;
+  avax_tx_enabled: boolean;
```

Comment:
smol nit but i feel like this should be `avalanche_mainnet_rpc`
```suggestion
  avalanche_tx_enabled: boolean;
```